src/Catty/ViewController/CatrobatTableViewController/FormTableViewController/FormSwitchTableViewCell.swift

```diff
@@ -25,7 +25,7 @@ import UIKit
 
 class FormSwitchTableViewCell: FormTableViewCell {
 
-    private let switchControl: UISwitch = {
+    let switchControl: UISwitch = {
         let switchControl = UISwitch()
         switchControl.accessibilityIdentifier = "formSwitch"
```

Comment:
Maybe we should set the [onTintColor](https://developer.apple.com/documentation/uikit/uiswitch/1623687-ontintcolor) of the UISwitches to our .globalTint, for a bit more uniformity